blockscout/envs/common-frontend.env

```
@@ -26,3 +26,4 @@ NEXT_PUBLIC_AD_TEXT_PROVIDER=none
 NEXT_PUBLIC_FEATURED_NETWORKS=
 NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID=0bcd4c7ef4545c3ed95149192d4d06aa
 NEXT_PUBLIC_OG_IMAGE_URL=file:///app/public/static/kadena_evm_og_image.png
+NEXT_PUBLIC_VIEWS_BLOCK_HIDDEN_FIELDS=[\"total_reward\"]
```

you dont need to scape `"`
```suggestion
NEXT_PUBLIC_VIEWS_BLOCK_HIDDEN_FIELDS=["total_reward"]
```